CoffeeChat/Controller/OnboardingPageViewController.swift

```diff
@@ -35,12 +53,18 @@ class OnboardingPageViewController: UIPageViewController {
     }
 }
 
-extension OnboardingPageViewController: OnboardingPageDelegate {
+extension OnboardingPageViewController: OnboardingPageDelegate, UIScrollViewDelegate {
     func nextPage(index: Int) {
         setViewControllers([onboardingPages[index]], direction: .forward, animated: true, completion: nil)
     }
 
     func backPage(index: Int) {
         setViewControllers([onboardingPages[index]], direction: .reverse, animated: true, completion: nil)
     }
+
+    func scrollViewDidScroll(_ scrollView: UIScrollView) {
+        // Update x position based on screen size when page scrolls
+        backgroundXPosition -= 0.04 * screenWidth
```

Comment:
wheres the `0.04` coming from?

Reply:
I printed out how often scrollViewDidScroll gets called and it seems to be after the page has scrolled 3% - 4% of the screen so I just used that value cause I want the background to scroll the same amount as the page.